sdk/textanalytics/ai-text-analytics/samples/javascript/README.md

```diff
@@ -0,0 +1,71 @@
+urlFragment: ai-text-analytics-javascript
+---
+
+# Azure Text Analytics client library samples for JavaScript
+
+These sample programs show how to use the JavaScript client libraries for Azure Cognitive Services Text Analytics in some common scenarios.
+
+| **File Name**                                         | **Description**                                                                       |
+| ----------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| [analyzeSentiment.js][analyzesentiment]               | analyzes the sentiment of a piece of text                                             |
+| [recognizePii.js][recognizepii]                       | detects personally-identifiable information                                           |
+| [detectLanguages.js][detectlanguages]                 | detects the language of a piece of text                                               |
+| [extractKeyPhrases.js][extractkeyphrases]             | extracts key phrases from a piece of text                                             |
+| [recognizeLinkedEntities.js][recognizelinkedentities] | detects entities that have links to more information on the web                       |
+| [tokenAuth.js][tokenauth]                             | uses Azure Active Directory (RBAC) to authenticate to the Cognitive Services endpoint |
+| [recognizeEntities.js][recognizeentities]             | detects entites in a piece of text and prints them along with the entity type         |
+
+## Prerequisites
+
+The samples are compatible with Node.js >= 8.0.0.
+
+You need [an Azure subscription][freesub] and [an Azure Cognitive Services Instance][azcogsvc] to run these sample programs. Samples retrieve credentials to access the Cognitive Services endpoint from environment variables. Alternatively, edit the source code to include the appropriate credentials. See each individual sample for details on which environment variables/credentials it requires to function.
+
+Adapting the samples to run in the browser may require some additional consideration. For details, please see the [package README][package].
+
+## Setup
+
+To run the samples using the published version of the package:
+
+1. Install the dependencies using `npm`:
+
+```bash
+npm install
+```
+
+2. Edit the file `sample.env`, adding the correct credentials to access the Azure service and run the samples. Then rename the file from `sample.env` to just `.env`. The sample programs will read this file automatically.
+
+3. Run whichever samples you like (note that some samples may require additional setup, see the table above):
+
+```bash
+node dist/basic.js
+```
+
+Alternatively, run a single sample with the correct environment variables set (step 3 is not required if you do this), for example (cross-platform):
+
+```bash
+npx cross-env ACCOUNT_NAME="<account name>" ACCOUNT_KEY="<account key>" node dist/basic.js
```

Comment:
`ACCOUNT_NAME` and `ACCOUNT_KEY` should be `COGNITIVE_SERVICES_ENDPOINT` and `SUBSCRIPTION_KEY` right?

Minor additional note that we're likely going to be scrubbing cognitive services from vars, so maybe this should be `TEXT_ANALYTICS_ENDPOINT` in samples?

Reply:
Yeah I forgot to convert this section of the READMEs here. There's a lot of manual intervention involved in porting these from package to package and I always forget _something_.

Reply:
Maybe you left it there on purpose to make me feel good about adding value in code review? ;)

Reply:
re: the naming what about just `API_ENDPOINT` or `ENDPOINT`? Just "ENDPOINT" is the label used in Azure Portal.

Reply:
I'm fine with `ENDPOINT`, though that's almost generic enough to be used by something else.